### PR TITLE
all tascs homework 5

### PR DESCRIPTION
**Pull request by AlexandrBelan**
Everything works locally

### CODE REVIEW

File: JS_lesson5/Task3.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        newIndex = 0;
+    }
+    if (str[i] !== alphabet[index]){
+        resultStr = resultStr + alphabet[newIndex].toUpperCase();
+    } else {
+        index = index + offset;
+        resultStr = resultStr + alphabet[newIndex];
+    }
+}
+console.log(resultStr)
```
Comment (VladimirLQA) [inline JS_lesson5/Task3.js]:
```suggestion
for (let i = 0; i < str.length; i++) {
    index = alphabet.indexOf(str[i].toLowerCase())
    newIndex = index + offset;
    if (index === 0 && offset < 0) {
        newIndex = alphabet.length -1;
    }
    if (index === alphabet.length -1 && offset > 0) {
        newIndex = 0;
    }
    if (str[i] !== alphabet[index]){
        resultStr +=  alphabet[newIndex].toUpperCase();
    } else {
        index += offset;
        resultStr += alphabet[newIndex];
    }
}
console.log(resultStr)
```

File: JS_lesson5/Task2.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        size = 'Gb'
+        break;
+    case 4:
+        size = 'Tb'
+        break; 
+}   
+console.log(`${byte} = ${result.toFixed(1)} ${size}`);
+
+
+// 2. Сделать из "*" в консоли равнобедренный треугольник и ромб
```
Comment (VladimirLQA) [inline JS_lesson5/Task2.js:45]: 2 in 1 ? :)
Need left side of diamond )
image

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

File: JS_lesson5/Task2.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+//   454 548 = 454,5 Kb
+//   1 454 548 = 1,5 Mb
+
+//   Результат должен быть округлен до 1 знака после запятой методом .toFixed(), про который надо почитать самим ;)
+
+const byte = 10000000000000;
+let size = '';
+let result = byte;
+let i = 0
+while (true) {
```
Comment (anatoly-karpovich) [inline JS_lesson5/Task2.js]: It is an infinity loop. Need a clear way to break a loop

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: JS_lesson5/Task3.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+let splitWord = word.toLowerCase().split('');
+let consonantsNum = 0;
+let containsNum = 0;
+for (let i = 0; i < word.length; i++) {
+    if (consonants.includes(splitWord[i])) {
+        consonantsNum++;
+    } else {
+        containsNum++;
+    }
+}
```
Comment (VladimirLQA) [inline JS_lesson5/Task3.js]:
```suggestion
const word = 'Wummermustdie';
const consonants = 'qwrtpsdfghjklzxcvbnm';
const vowels = 'eyuioa';
let splitWord = word.toLowerCase().split('');
let countConsonants = 0;
let countVowels = 0;
for (let i = 0; i < word.length; i++) {
    if (consonants.includes(splitWord[i])) {
        countConsonants++;
    } else {
        countVowels++;
    }
}
```

File: JS_lesson5/Task3.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+// console.log(`word contains ${containsNum} vowels and ${consonantsNum} consonants`);
```
Comment (VladimirLQA) [inline JS_lesson5/Task3.js]: Try to test this example `ab21`

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

File: JS_lesson5/Task3.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+
+// 2**. Написать программу, которая видоизменяет принимаемое слово (переменная со словом) 
+//   шифром ЦЕЗАРЯ (посмотреть в википедии) со сдвигом на 1 в любую из сторон. 
+//   Направление шифрования задается переменной offset, которая может быть +1 и -1.
+//   Например let str = 'AbC'; let offset = -1, result = 'ZaB';
+//   Например let str = 'ZzZ'; let offset = 1, result = 'AaA';
+const alphabet = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
+const str = 'abcZ';
```
Comment (VladimirLQA) [inline JS_lesson5/Task3.js]: Try to modify this solution to handle such case `ab 1Z` - leave space and number in the result line

**Review by anatoly-karpovich — APPROVED** [review]

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**